GVFS/GVFS.Platform.Windows/ProjFSFilter.cs

```diff
@@ -320,18 +321,34 @@ public string FlushDriverLogs()
             return sb.ToString();
         }
 
-        public bool TryPrepareFolderForCallbacks(string folderPath, out string error)
+        public bool TryPrepareFolderForCallbacks(string folderPath, out string error, out Exception exception)

```

Comment:
Minor: both of the out params are only used for tracing errors. Should this method just take an ITracer as an input param and writes out its own errors?

Reply:
> both of the out params are only used for tracing errors

@sanoursa, the `exception` parameter is only used for tracing, but the `error` out param is used for both tracing and for letter the user know (on the command line) why their clone as failed.

I can update `TryPrepareFolderForCallbacks` to take an `ITracer` (and remove the out `exception`), but we'd still need an out `error` parameter.  

Let me know if you think it's worth making that change.

Reply:
Discussed with @sanoursa offline and I will merge this in as-is.